docs/conf.py

```diff
@@ -14,6 +14,7 @@
 autodoc_mock_imports = ["f90nml", "iotaa", "jsonschema", "lxml", "referencing"]
 autodoc_typehints = "description"
 copyright = str(dt.datetime.now().year)
+exclude_patterns = ["**/shared/*.rst"]
```

Comment:
This prevents Sphinx from trying to process fragment `.rst` files in `shared/` directories.